pyOCD/transport/cmsis_dap.py

```diff
@@ -65,6 +65,12 @@
                  32: CSW_SIZE32
                  }
 
+DAP_MODE_SWD = 1
+DAP_MODE_JTAG = 2
+
+STICKORUN = 0x00000002
```

Comment:
Shall we add prefix, this is a group of errors. Something like above DAP_MODE_xxxx